cPP_DSC.adoc

```diff
@@ -2968,7 +3016,7 @@ _Application Note {counter:remark_count}_:: _It is acceptable for the TSF to pro
 
 FTP_CCMP_EXT.1 CCM Protocol
 
-FTP_CCMP_EXT.1.1:: The TSF shall implement CCMP using [.underline]#[selection: AES, Camellia]# in CCM mode and key size [.underline]#[selection: 128-bits, 256-bits]# as defined in [.underline]#[selection: IEEE 802.11i, IEEE 802.11ac]#.
```

Comment:
IEEE 802.11i says "CCMP is based on the CCM of the AES encryption algorithm". IEEE 802.11ac says "The AES algorithm is defined in FIPS PUB 197-2001. AES processing used within CCMP uses AES with either a 128-bit key (CCMP-128) or a 256-bit key (CCMP-256)."

Why is Camellia allowed here? Is there any use case?